src/ParallelAlgorithms/LinearSolver/ConjugateGradient/InitializeElement.hpp

```diff
@@ -51,24 +58,18 @@ struct InitializeElement {
                     const ArrayIndex& /*array_index*/,
                     const ActionList /*meta*/,
                     const ParallelComponent* const /*meta*/) noexcept {
-    return std::make_tuple(
-        ::Initialization::merge_into_databox<
-            InitializeElement,
-            db::AddSimpleTags<Convergence::Tags::IterationId<OptionsGroup>,
-                              operator_applied_to_fields_tag, operand_tag,
-                              operator_applied_to_operand_tag, residual_tag,
-                              Convergence::Tags::HasConverged<OptionsGroup>>>(
-            std::move(box),
-            // The `PrepareSolve` action populates these tags with initial
-            // values, except for `operator_applied_to_fields_tag` which is
-            // expected to be filled at that point and
-            // `operator_applied_to_operand_tag` which is expected to be updated
-            // in every iteration of the algorithm.
-            std::numeric_limits<size_t>::max(),
-            typename operator_applied_to_fields_tag::type{},
-            typename operand_tag::type{},
-            typename operator_applied_to_operand_tag::type{},
-            typename residual_tag::type{}, Convergence::HasConverged{}));
+    // The `PrepareSolve` action populates these tags with initial
+    // values, except for `operator_applied_to_fields_tag` which is
+    // expected to be filled at that point and
+    // `operator_applied_to_operand_tag` which is expected to be updated
+    // in every iteration of the algorithm.
+    Initialization::mutate_assign<simple_tags>(
+        make_not_null(&box), std::numeric_limits<size_t>::max(),
+        typename operator_applied_to_fields_tag::type{},
+        typename operand_tag::type{},
+        typename operator_applied_to_operand_tag::type{},
+        typename residual_tag::type{}, Convergence::HasConverged{});
```

Comment:
Please delete these default-constructions, as you did in `AsynchronousSolvers/ElementActions.hpp`